### PR TITLE
Clean up, export some of the new helper subs and document them.

### PR DESCRIPTION
**Pull request by thundergnat**

### CODE REVIEW

File: lib/Math/Sequences/Integer.pm6 [hunk 1]
```diff
@@ -328,11 +328,12 @@ sub FatPi($digits=100) is export {
   FatRat.new(+([~] Pi-digits[^($digits)]), 10**($digits-1));
 }
 
-sub Eulers-number ( Int $terms = 264 ) {
-    # Generates approximately 1.9 accurate digits of e per term.
-    # Returns first 500 digits by default as a trade-off
+sub Eulers-numbera ( Int $terms = 500 ) is export(:support) {
```
Comment (AlexDaniel) [inline lib/Math/Sequences/Integer.pm6]: `numbera`?

File: lib/Math/Sequences/Integer.pm6 [hunk 2]
```diff
@@ -328,11 +328,12 @@ sub FatPi($digits=100) is export {
...
+    # Generates decimal digits of e acurate at least up to term.
```
Comment (AlexDaniel) [inline lib/Math/Sequences/Integer.pm6]: Small type: ac**c**urate

**Comment by thundergnat** [comment]
Sigh. I wonder how I mange to get dressed in the mornings sometimes.

**Comment by AlexDaniel** [comment]
@thundergnat actually, now I see there's a problem:
```
===SORRY!=== Error while compiling /home/alex/git/perl6-Math-Sequences/lib/Math/Sequences/Integer.pm6 (Math::Sequences::Integer)
Missing block
at /home/alex/git/perl6-Math-Sequences/lib/Math/Sequences/Integer.pm6 (Math::Sequences::Integer):293
------> sub totient ($n) is export⏏ (:support) {
    expecting any of:
        new name to be defined
```

**Comment by taboege** [comment]
I have a habit of leaving a space between function name and signature. (That's how I imagine this typo came to be.) That space is not allowed after the `export` trait, simple as that.

**Pull request merged**